lib/evaluate.js

```diff
@@ -294,35 +296,32 @@ export const runFraudDetection = async (roundIndex, measurements, sparkRoundDeta
     // 2. Pick the measurement with the lowest hash value
     // This relies on the fact that the hash function has a random distribution.
     // We are also making the assumption that each measurement has a distinct `finished_at` field.
+    //
+    const measurementsWithHash = await Promise.all(
+      groupMeasurements.map(async (m) => ({ m, h: await getHash(m) }))
+    )
+    // Sort measurements using the computed hash
+    measurementsWithHash.sort((a, b) => a.h < b.h ? -1 : a.h > b.h ? 1 : 0)
```

Comment:
```suggestion
    measurementsWithHash.sort((a, b) => a.h.localeCompare(b.h))
```

Reply:
I am a bit concerned about what locale will be used for comparison. I think it's fine to assume the Fly.io environment has a reasonable locale, but how about local development? I don't want the tests to fail if my locale is set to CZ or DE.

Fortunately, we can hard-code the locale in the second argument.

```js
a.h.localeCompare(b.h, 'en')
```

Reply:
Addressed in [668ebf2](https://github.com/filecoin-station/spark-evaluate/pull/225/commits/668ebf2a2ba02414da795de84b93650631a32ff3)